Source/Fuse.Scripting.JavaScript/ObservableProperty.uno

```diff
@@ -144,6 +236,8 @@ namespace Fuse.Scripting.JavaScript
 
 		void PushValue(Scripting.Context context, object val)
 		{
+			if (_subscription == null) return;
+
```

Comment:
This looks like a general bug-fix, and could have been done separately on master.

Reply:
I've separate this into #811